### ISSUE TITLE
Access denied: Try using the correct credentials.

### ISSUE THREAD

**Issue by shawnbro** [plaid/plaid-ruby#50]
Whenever I try to add a user with real credentials I get the above error message.  Not sure why this is happening & it was working a few days ago.  Has something changed in the last view days regarding the API?

UPDATE: a few of us have tried logging in with our credentials with the same result, so I don't think it's an issue with inputting the wrong username/password.  The request looks like this: `Plaid.add_user('auth', 'username', 'password', 'citi')`.  Also - environment location is set to `'https://tartan.plaid.com/'`

UPDATE 2: I believe I figured out the issue - I've added too many accounts to my test account.  Shouldn't the error message be more responsive?

**Comment by michaelckelly** [comment]
@shawnbro 

If you've exceeded the maximum number of additions, the API will respond with a `code 1112 / addition limit exceeded` error ([error code reference](https://github.com/plaid/support/blob/master/errors.md/)). 

The issue here might be that the ruby client library is not returning the entire response body as the error.  Do you mind sharing the entire response?

**Comment by shawnbro** [comment]
@michaelckelly Thanks for your response.

When I make the request with real credentials, the only response I get is `RuntimeError: Access denied: Try using the correct credentials.` from `/lib/plaid/util.rb:55`. 

When I looked at the source I noticed that it throws that error for `err` of `Unauthorized`.  I switched the gem usage to another API ('zen-plaid') - and retried the request and received the code 1112 / addition limit exceeded error, which is what led me to believe that I had exceeded my limit & close the issue.

I'd like to stick to the official Ruby wrapper for the API, and I'm in the process of getting our account upgraded past pending status.  Is there any way to delete all the user's associated with the account if I don't have access tokens? Or use the website to retrieve the access tokens?  (we've been testing and our data has gotten cleared).
